test/acceptance/test/utils.go

```diff
@@ -330,12 +330,12 @@ func runWegoAddCommandWithOutput(repoAbsolutePath string, addCommand string, weg
 	return string(session.Wait().Out.Contents()), string(session.Wait().Err.Contents())
 }
 
-func runWegoAddCommandAndReturnSession(repoAbsolutePath string, addCommand string, wegoNamespace string) *gexec.Session {
-	command := exec.Command("sh", "-c", fmt.Sprintf("cd %s && %s %s", repoAbsolutePath, WEGO_BIN_PATH, addCommand))
-	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
-	Expect(err).ShouldNot(HaveOccurred())
-	return session
-}
+// func runWegoAddCommandAndReturnSession(repoAbsolutePath string, addCommand string, wegoNamespace string) *gexec.Session {
```

Comment:
Same here. Why is this commented out?

Reply:
A useful function I could probably get rid of, for now